lib/Model/Acl.php

```diff
@@ -489,4 +443,170 @@ private function getIsShareValidForUsers(): bool {
 	private function getHasEmail(): bool {
 		return $this->share->getToken() ? strlen($this->share->getEmailAddress()) > 0 : $this->getIsLoggedIn();
 	}
+
+	/**
+	 * Checks, if user is allowed to edit the poll configuration
+	 **/
+	private function getAllowEditPoll(): bool {
+		// Console god mode
+		if (defined('OC_CONSOLE')) {
```

Comment:
There is also `OC::$CLI`
But that is also true for system cron jobs.

Reply:
I try to avoid OC if possible, since it is not defined on dev time.